editor/src/core/model/conditionals.ts

```diff
@@ -16,24 +14,13 @@ import { findUtopiaCommentFlag, isUtopiaCommentFlagConditional } from '../shared
 
 export type ConditionalCase = 'true-case' | 'false-case'
 
-export function getConditionalCasePath(
-  elementPath: ElementPath,
-  conditionalCase: ConditionalCase,
-): ElementPath {
-  return EP.appendToPath(elementPath, conditionalCase)
-}
-
 // Get the path for the clause (true case or false case) of a conditional.
 export function getConditionalClausePath(
   conditionalPath: ElementPath,
-  conditionalClause: ChildOrAttribute,
+  conditionalClause: JSXElementChild,
   conditionalCase: ConditionalCase,
```

Comment:
I think with this PR merged we should make a quick follow-up PR that erases the concept of the ConditionalClausePath